contracts/AssetPool.sol

```diff
@@ -29,16 +31,19 @@ import "./UnderwriterToken.sol";
 ///         in KEEP in return for covKEEP underwriter tokens. Underwriter tokens
 ///         represent an ownership share in the underlying collateral of the
 ///         Asset Pool.
-contract AssetPool is Ownable {
+contract AssetPool is Ownable, IAssetPool {
     using SafeERC20 for IERC20;
     using SafeERC20 for UnderwriterToken;
     using SafeMath for uint256;
 
     IERC20 public collateralToken;
     UnderwriterToken public underwriterToken;
+    UnderwriterToken public newUnderwriterToken;
 
     RewardsPool public rewardsPool;
 
+    IAssetPoolUpgrade public newAssetPool;
```

Comment:
This is no longer needed, right?

Reply:
[here](https://github.com/keep-network/coverage-pools/blob/upgradeability-for-asset-pool/contracts/AssetPool.sol#L322) we call `depositFor()` on a new Asset Pool. Before we can call it, the governance has to [approve](https://github.com/keep-network/coverage-pools/blob/upgradeability-for-asset-pool/contracts/AssetPool.sol#L347) it. We need `newAssetPool` if we do it this way.

Reply:
Oops I am sorry, GitHub tricked me, I probably did not refresh the page before posting this comment. This comment was for line 41 with `newUnderwritertToken`.

Reply:
Ah, right `newUnderwriterToken` is not needed. Removed f757e4622d79e20c4e7c1e07c1530d407fab334a